libc/vga/vga.h

```diff
@@ -0,0 +1,15 @@
+#ifndef COSMOPOLITAN_LIBC_VGA_VGA_H_
```

Comment:
Please rename this header `vga.internal.h` so it doesn't end up in cosmopolitan.h. We'll likely want `LIBC_VGA` to be part of the amalgamation, as defined by `Makefile`.

Reply:
> Please rename this header `vga.internal.h` so it doesn't end up in cosmopolitan.h.

I have done so.  Thanks!